docs/ru/operations/settings/settings.md

```diff
@@ -85,6 +85,21 @@ ClickHouse применяет настройку в тех случаях, ко
 
 Имеет смысл выключать, если на сервере миллионы мелких таблиц-чанков, которые постоянно создаются и уничтожаются.
 
+## function_range_max_elements_in_block {#settings-function_range_max_elements_in_block}
+
+Устанавливает порог безопасности для объема данных, создаваемого функцией [range](../../sql-reference/functions/array-functions.md#range). Задаёт максимальное количество значений, генерируемых функцией на блок данных (сумма размеров массивов для каждой строки в блоке).
+
+Возможные значения:
+
+-   Положительное целое.
```

Comment:
```suggestion
-   Положительное целое число.
```

Reply:
Anyway, one suggestion here (out of three) is not about style (about translating the word 'integer') -- please confirm.